# ECES450650-Tutorial7: Reviewing how to use a metagenomic assembler and the metrics involved in assessing its performance. 
Goal: compare metrics in the assembler, such as N50, aberage contigs' size, etc. using QUAST (https://kbase.us/applist/apps/kb_quast/run_QUAST_app/release)

## Tool: 
KBase (https://kbase.us/n/33233/351/)
Assembly and annotation documentation: https://docs.kbase.us/apps/analysis/assembly-and-annotation

## Dataset: 
/ifs/groups/eces450650Grp/data/CAMI_low_RL_S001__insert_270_GoldStandardAssembly.fasta.gz from Picotte

## Team Members:
* Bethany Pittman
* Neil Patel
* Tobenna Oduah

## Kbase Step-By-Step Instructions with QUAST
1. Sign up/Login at https://www.kbase.us/.
2. Click "New Narrative."
3. Click "Add Data" in the Data column on the left.
4. Import from URL or local computer
5. select file format/unzip if necessary (can also unzip using the "DATA" app).
6. Run imported data.
7. Search for and select "QUAST" in the App column on the left.
8. Add file path/object.
9. Submit job.

### QUAST Report on Data Attached: 
https://kbase.us/dynserv/66e9a37e7d3a56b64cc372c89d1eaafaa605c397.HTMLFileSetServ/api/v1/116626/6/1/$/0/report.html
